frontend/src/pages/org/archived-items.ts

```diff
@@ -527,12 +540,16 @@ export class CrawlsList extends LiteElement {
         () => html`
           <sl-menu-item
             @click=${() =>
-              this.navTo(`${this.orgBasePath}/workflows/crawl/${item.cid}`)}
+              this.navTo(
+                `${this.orgBasePath}/workflows/crawl/${(item as Crawl).cid}`
```

Comment:
I'm pretty sure we could drop the type coercion if we swap out the `when` expression for an `&&` — Typescript knows enough to figure out that `item` is a `Crawl` when `item.type === "crawl"`, but it doesn't know how to do that within a `when` expression

Reply:
Good call!